### PR TITLE
[MB-15646] Add spreadsheet file types to gitignore prior to pricing import work

### PR DESCRIPTION
**Pull request by duncan-truss**
## Summary

This adds a bunch of spreadsheet type file extensions to our gitignore to avoid accidentally committing what could be sensitive pricing data.

You could still add these file types but you would either need to explicitly allow them in the .gitignore by adding a `!` in front or by using the force option of git add.

Screenshot 2023-04-18 at 1 43 58 PM

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

### Additional steps

1. Access the
2. Login as a
3.

## Verification Steps for Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Request review from a member of a different team.
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Frontend

- [ ] User facing changes have been reviewed by design.
- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://github.com/transcom/mymove/tree/main/docs/adr/0016-Browser-Support.md) (Chrome, Firefox, IE, Edge).
- [ ] There are no new console errors in the browser devtools
- [ ] There are no new console errors in the test output
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide#logging)
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/dev/contributing/backend/Backend-Programming-Guide/#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#zero-downtime-migrations)
- [ ] Have been communicated to #g-database
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/dev/contributing/database/Database-Migrations#secure-migrations)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -66,6 +66,39 @@ playwright/results
...
+!pkg/parser/transittime/fixtures/Appendix_C(i)_-_Transit_Time_Tables_Fake_Data.xlsx
+!pkg/testdatagen/testdata/Weight Estimator.xlsx
+!pkg/testdatagen/testdata/Weight Estimator.xls
+!pkg/testdatagen/testdata/test.xls
+!playwright/fixtures/constructedWeight.xlsx
+!playwright/fixtures/constructedWeight.xls
+!src/components/DocumentViewer/sample.xls
+
 # XLSX backup files
 ~$*.xlsx
```
Comment (reggieriser) [inline .gitignore:103]: **nitpick:** Would this rule already be covered by your more broad `*.xls*` rule now?  Shouldn't hurt anything, though.

**Review by reggieriser — APPROVED** [review]
LGTM!  I tried copying an *.xlsx, *.xls, and *.csv file to the top directory and none showed up in a `git status` as desired.  I also tried editing one of the explicitly included files and it correctly showed it as a mod.

**Review by PDickeyTruss — APPROVED** [review]
LGTM 🚢

File: .gitignore (hunk 1 above)
Comment (duncan-truss) [reply .gitignore:103]: Good point I think you're right. I guess I wasn't completely sure if the `~` or `$` had special significance in the gitignore like `!`.  I also don't know how to recreate the backup or recovery excel file locally.

**Pull request merged**